test/cli/run.spec.ts

```diff
@@ -186,7 +189,14 @@ async function exec(opts: Options = {}) {
   }
 
   if (outputString === null) {
```

Comment:
This check is still always `true`, because `outputString` doesn't change after declaring at line 126